src/app/compiler.js

```diff
@@ -180,15 +180,15 @@ function Compiler (editor, handleGithubCall, outputField, hidingRHP, updateFiles
       }
       while (importHints.length > 0) {
         var m = importHints.pop();
+        if (m.startsWith('./')) {
```

Comment:
Ah right, now I see it. Please move that to after the regex match.
Actually we should not do regex searches if we have a compiler that supports the import callback.